### ISSUE TITLE
Unknown US/Pacific timezone

### ISSUE THREAD

**Issue by amelchio** [iCalcreator/iCalcreator#114]
I am not entirely sure whether this is a bug or maybe I need to enable additional timezones somehow? Changing the TZID to `America/Los_Angeles` works without error.

# Test program

```
<?php

$ical = <<<EOF
BEGIN:VCALENDAR
VERSION:2.0
X-APPLE-CALENDAR-COLOR:#FF2968

BEGIN:VEVENT
DTSTART;TZID=US/Pacific:20170408T120000
DTEND;TZID=US/Pacific:20170408T150000
END:VEVENT

END:VCALENDAR
EOF;

require_once('/iCalcreator/autoload.php');

$cal = new Kigkonsult\Icalcreator\Vcalendar();
$cal->parse($ical);

$cal->selectComponents(2017, 4, 1, 2017, 4, 30);
```

# Output

```
PHP Fatal error:  Uncaught Exception: Failed to parse time string (2017-04-08 12:00:00 US/Pacific) at position 20 (U): The timezone could not be found in the database in /iCalcreator/src/Util/UtilDateTime.php:78
Stack trace:
#0 /iCalcreator/src/Util/UtilDateTime.php(78): DateTime->__construct()
#1 /iCalcreator/src/Util/UtilDateTime.php(204): Kigkonsult\Icalcreator\Util\UtilDateTime->__construct()
#2 /iCalcreator/src/Util/SelectFactory.php(183): Kigkonsult\Icalcreator\Util\UtilDateTime::factory()
#3 /iCalcreator/src/Vcalendar.php(527): Kigkonsult\Icalcreator\Util\SelectFactory::selectComponents()
#4 /bugs/pacific.php(21): Kigkonsult\Icalcreator\Vcalendar->selectComponents()
#5 {main}

Next RuntimeException: #3 Can't create (to-)DateTime : '2017-04-08 12:00:00 US/Pacific' in /iCalcreator/src/Util/UtilDateTime.php:207
Stack trace:
#0 /iCalcreator/src/Util/SelectFactory.php(183): Kigkonsult\Icalcreator\Util\UtilDateTime::factory()
#1 /iCalcreator/src/Vcalendar.php(527): Kigkonsult\Icalcreator\Util\SelectFactory::selectComponents()
#2 /bugs/pacific.php(21): Kigkonsult\Icalcreator\Vcalendar->selectComponents()
#3 {main}
  thrown in /iCalcreator/src/Util/UtilDateTime.php on line 207
```

**Comment by iCalcreator** [comment]
Fixed in the 2.41.83 release

**Comment by amelchio** [comment]
Verified 👍. Thank you.
